README.md

```diff
@@ -1,8 +1,14 @@
 # Wikimedia Commons Android app
-![GitHub issue custom search](https://img.shields.io/github/issues-search?label=%22good%20first%20issue%22%20issues&query=repo%3Acommons-app%2Fapps-android-commons%20is%3Aissue%20is%3Aopen%20label%3A%22good%20first%20issue%22)
-[![Build status](https://github.com/commons-app/apps-android-commons/actions/workflows/android.yml/badge.svg?branch=master)](https://github.com/commons-app/apps-android-commons/actions?query=branch%3Amaster)
+
+[![Build status](https://github.com/commons-app/apps-android-commons/actions/workflows/android.yml/badge.svg?branch=main)](https://github.com/commons-app/apps-android-commons/actions?query=branch%3Amain)
 [![Preview the app](https://img.shields.io/badge/Preview-Appetize.io-orange.svg)](https://appetize.io/app/8ywtpe9f8tb8h6bey11c92vkcw)
 [![codecov](https://codecov.io/gh/commons-app/apps-android-commons/branch/master/graph/badge.svg)](https://codecov.io/gh/commons-app/apps-android-commons)
+[![Good First Issues](https://img.shields.io/github/issues-search?label=%22good%20first%20issue%22&query=repo%3Acommons-app%2Fapps-android-commons%20is%3Aissue%20is%3Aopen%20label%3A%22good%20first%20issue%22)](https://github.com/commons-app/apps-android-commons/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)
+[![Bug Issues](https://img.shields.io/github/issues-search?label=%22bug%22&query=repo%3Acommons-app%2Fapps-android-commons%20is%3Aissue%20is%3Aopen%20label%3A%22bug%22)](https://github.com/commons-app/apps-android-commons/issues?q=is%3Aissue+is%3Aopen+label%3A%22bug%22)
+[![Enhancement Issues](https://img.shields.io/github/issues-search?label=%22enhancement%22&query=repo%3Acommons-app%2Fapps-android-commons%20is%3Aissue%20is%3Aopen%20label%3A%22enhancement%22)](https://github.com/commons-app/apps-android-commons/issues?q=is%3Aissue+is%3Aopen+label%3A%22enhancement%22)
+[![Open Issues](https://img.shields.io/github/issues/commons-app/apps-android-commons)](https://github.com/commons-app/apps-android-commons/issues)
+[![Assigned Issues](https://img.shields.io/github/issues-search?label=%22Assigned%20Issues%22&query=repo%3Acommons-app%2Fapps-android-commons%20is%3Aissue%20is%3Aopen%20assignee%3A%2A)](https://github.com/commons-app/apps-android-commons/issues?q=is%3Aissue+is%3Aopen+assignee%3A%2A)
+[![Unassigned Issues](https://img.shields.io/github/issues-search?label=%22Unassigned%20Issues%22&query=repo%3Acommons-app%2Fapps-android-commons%20is%3Aissue%20is%3Aopen%20no%3Aassignee)](https://github.com/commons-app/apps-android-commons/issues?q=is%3Aissue+is%3Aopen+no%3Aassignee)
```

Comment:
I think we already have the important links:
![Screenshot_20250224-105733~2.png](https://github.com/user-attachments/assets/aeaf40cc-9dea-47f4-b20a-057ed2654ac3)

I changed the second to point to the best bugs for newcomers.